test/functional/ui/screen.lua

```diff
@@ -205,6 +205,8 @@ end
 
 function Screen:try_resize(columns, rows)
   uimeths.try_resize(columns, rows)
+  -- Give ourselves a chance to _handle_resize
+  self:sleep(0.1)
```

Comment:
`helpers.retry()` may need to somehow call `self:sleep(0.001)` on each iteration. Hmm.